packages/nc-gui/components/smartsheet/expanded-form/Header.vue

```diff
@@ -25,6 +25,8 @@ const { isUIAllowed } = useUIPermission()
 
 const reloadTrigger = inject(ReloadRowDataHookInj, createEventHook())
 
+const saveRowAndStay = ref(0)
```

Comment:
We can persist the state in localstorage

Reply:
Agree. May do it later.